docs/product-features/staking/key-concepts.md

```diff
@@ -1,3 +1,55 @@
 # Key concepts
 
-Comming soon.
\ No newline at end of file
+## Staking Basics
+
+Staking consists of locking your ALEX tokens temporarily to secure the network and earn rewards in return. In a Proof of Stake (PoS) consensus mechanism, such as the one used by ALEX Lab, validators stake their tokens as collateral and earn rewards in the form of newly minted tokens and transaction fees. 
+
+The ALEX staking pool offers two main options, each one rewarding stakers with a different token: manual staking with $ALEX or liquid staking with LiALEX. 
+
+### Proof of Stake (PoS)
+
+Proof of Stake (PoS) is a consensus mechanism for processing transactions and securing a blockchain network. Unlike Proof of Work (PoW), which requires miners to expend computational effort to solve a cryprographic puzzle, PoS selects validators based on their stake in the network: the larger their stake, the more likely they are to be selected for validation. 
+
+The ALEX Lab network is secured by validators and delegators. 
+
+* **Validators:**  They are node operators that verify transactions and create new blocks while staking their own $ALEX. The stake acts as collateral, encouraging validators to behave honestly, since they risk losing their staked tokens if they validate fraudulent blocks.
```

Comment:
no validator as no block production... etc

Reply:
Discarded this section (at least temporarily)